Tests/XCUITests/PhotonActionSheetTest.swift

```diff
@@ -3,6 +3,7 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
 import XCTest
+import Common
```

Comment:
Could you please add `import Common` in `BaseTestCase` instead? This way we'd have all image references in all tests.

Reply:
@clarmso Had to revert the removal of the imports as the build was failing due to the imports missing.